halo2_proofs/src/poly/evaluator.rs

```diff
@@ -210,20 +210,38 @@ impl<E, F: Field, B: Basis> Evaluator<E, F, B> {
         // Apply `ast` to each chunk in parallel, writing the result into an output
         // polynomial.
         let mut result = B::empty_poly(domain);
-        multicore::scope(|scope| {
+        #[cfg(feature = "multicore")]
+        {
+            multicore::scope(|scope| {
+                for (chunk_index, out) in result.chunks_mut(chunk_size).enumerate() {
+                    scope.spawn(move |_| {
+                        let ctx = AstContext {
+                            domain,
+                            poly_len,
+                            chunk_size,
+                            chunk_index,
+                            polys: &self.polys,
+                        };
+                        out.copy_from_slice(&recurse(ast, &ctx));
+                    });
+                }
+            });
+        }
+
```

Comment:
```suggestion
```

Reply:
Thanks for fixing the typos.  Will apply.